README.md

```diff
@@ -1,54 +1,65 @@
-# [`<INSERT MODULE NAME>` module](<INSERT LINK TO MODULE REPO>)
+# youtubeStream modular service
 
-This [module](https://docs.viam.com/registry/#modular-resources) implements the [`<INSERT API TRIPLET>` API]<INSERT LINK TO DOCS (if applicable)> in an <INSERT MODEL> model.
-With this model, you can...
+This module implements the [rdk camera API](https://github.com/rdk/camera-api) in a youtubeStream model.
+With this model, you can provide a youtube URL as a configurable attribute to a camera component.
 
 ## Requirements
 
-_Add instructions here for any requirements._
+The module executable is currently only supported on `darwin/arm64`. Make sure your machine is running on one of these architectures to avoid exec format issues.
```

Comment:
Your pr comment shows a working module on the raspberry pi, was that from a local build?

Reply:
That is just a typo on my end for not including linux/arm64 in the readMe. I will fix that